tools/tests/embed/embed_division.py

```diff
@@ -1,5 +1,4 @@
 """This tests that future compiler flags are passed to the embedded IPython."""
-from __future__ import division
```

Comment:
This file is also now not useful; it could also be adapted to use `barry_as_FLUFL`.